src/snowflake/cli/_plugins/spcs/services/commands.py

```diff
@@ -297,6 +298,162 @@ def logs(
     return StreamResult(cast(Generator[CommandResult, None, None], stream))
 
 
+@app.command(requires_connection=True)
+def events(
+    name: FQN = ServiceNameArgument,
+    container_name: str = typer.Option(
+        ...,
+        "--container-name",
+        help="Name of the container.",
+        show_default=False,
+    ),
+    instance_id: str = typer.Option(
+        ...,
+        "--instance-id",
+        help="ID of the service instance, starting with 0.",
+        show_default=False,
+    ),
+    since: str = typer.Option(
+        default="",
+        help="Fetch events that are newer than this time ago, in Snowflake interval syntax.",
+    ),
+    until: str = typer.Option(
+        default="",
+        help="Fetch events that are older than this time ago, in Snowflake interval syntax.",
+    ),
+    first: int = typer.Option(
+        default=-1,
+        show_default=False,
+        help="Fetch only the first N events. Cannot be used with --last.",
+    ),
+    last: int = typer.Option(
+        default=-1,
+        show_default=False,
+        help="Fetch only the last N events. Cannot be used with --first.",
+    ),
+    **options,
+):
+    if FeatureFlag.ENABLE_SPCS_SERVICE_EVENTS.is_disabled():
+        raise FeatureNotEnabledError(
+            "ENABLE_SPCS_SERVICE_EVENTS",
+            "Service events collection from SPCS event table is disabled.",
+        )
+
+    if first >= 0 and last >= 0:
+        raise IncompatibleParametersError(["--first", "--last"])
+
```

Comment:
can we add more checks here about timestamp? and throw message to customer on how timestamp should look like ?

Reply:
ok, I would do it today